src/index.js

```diff
@@ -7,116 +7,91 @@ const {
   log
 } = require('cozy-konnector-libs')
 const request = requestFactory({
-  // the debug mode shows all the details about http request and responses. Very usefull for
-  // debugging but very verbose. That is why it is commented out by default
-  // debug: true,
-  // activates [cheerio](https://cheerio.js.org/) parsing on each page
   cheerio: true,
-  // If cheerio is activated do not forget to deactivate json parsing (which is activated by
-  // default in cozy-konnector-libs
-  json: false,
-  // this allows request-promise to keep cookies between requests
+  debug: false,
   jar: true
 })
+const moment = require('moment')
 
-const baseUrl = 'http://books.toscrape.com'
+const baseUrl = 'https://www.glisshop.com'
 
 module.exports = new BaseKonnector(start)
 
-// The start function is run by the BaseKonnector instance only when it got all the account
-// information (fields). When you run this connector yourself in "standalone" mode or "dev" mode,
-// the account information come from ./konnector-dev-config.json file
 async function start(fields) {
   log('info', 'Authenticating ...')
   await authenticate(fields.login, fields.password)
   log('info', 'Successfully logged in')
-  // The BaseKonnector instance expects a Promise as return of the function
   log('info', 'Fetching the list of documents')
-  const $ = await request(`${baseUrl}/index.html`)
-  // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
+  const $ = await request(`${baseUrl}/mon_compte/mes_commandes/`)
   log('info', 'Parsing list of documents')
   const documents = await parseDocuments($)
-
-  // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // common case in connectors
+  log('info', `${documents.length} documents found `)
   log('info', 'Saving data to Cozy')
   await saveBills(documents, fields.folderPath, {
-    // this is a bank identifier which will be used to link bills to bank operations. These
-    // identifiers should be at least a word found in the title of a bank operation related to this
-    // bill. It is not case sensitive.
-    identifiers: ['books']
+    identifiers: ['glisshop'],
+    contentType: 'application/pdf'
```

Comment:
This specific option seems weird to me as I can't understand why `saveBills` needs a `contentType`. I know how the `saveBills` function is implemented and that this parameter is going to be passed to a `request/request` function. But still, we need to update the API. Not now, not within this PR, but someday, somehow.

Reply:
Yes probably, anyway, it works better with the viewer like that as you know so i keep it.